plugins/cpp_metrics/parser/src/cppmetricsparser.cpp

```diff
@@ -1,4 +1,5 @@
 #include <cppmetricsparser/cppmetricsparser.h>
+#include "efferent.h"
```

Comment:
Incorrect inclusion order. Please refer to the [coding conventions](https://github.com/Ericsson/CodeCompass/blob/master/doc/coding_conventions.md#headers).